requirements_dev.txt

```diff
@@ -20,3 +20,4 @@ Sphinx==1.4.8
 pytest==2.9.2
 pytest-cov
 codecov
+ipython
```

Comment:
since `requirements-dev.txt` is for the needed packages when developing, I added there. But I could remove it.